### PR TITLE
rollback block admin commands

### PR DESCRIPTION
**Pull request by charithabandi**

### CODE REVIEW

File: node/services/jsonrpc/adminsvc/service.go [hunk 1]
```diff
@@ -561,3 +571,45 @@ func (svc *Service) ResolutionStatus(ctx context.Context, req *adminjson.Resolut
...
+		txInfo[i] = &types.TxInfo{
+			ID:     txID,
+			Status: status.TxStatus[txID.String()],
+		}
+	}
+	resp.TxInfo = txInfo
+	return resp, nil
+}
+
+func (svc *Service) RollbackBlock(ctx context.Context, req *adminjson.RollbackBlockRequest) (*adminjson.RollbackBlockResponse, *jsonrpc.Error) {
```
Comment (jchappelow) [inline node/services/jsonrpc/adminsvc/service.go:598]: I feel like this RPC is a big misleading.  You can't rollback any arbitrary block.  We were referring to it as "reset" or "abort", referring to an uncommitted consensus round?  Thoughts?

File: node/consensus/engine.go [hunk 2]
```diff
@@ -332,7 +341,7 @@ func (ce *ConsensusEngine) runConsensusEventLoop(ctx context.Context) error {
 	ce.log.Info("Starting the consensus event loop...")
 	catchUpTicker := time.NewTicker(5 * time.Second)
 	reannounceTicker := time.NewTicker(3 * time.Second)
-	blkPropTicker := time.NewTicker(1 * time.Second)
+	blkPropTicker := time.NewTicker(10 * time.Second)
```
Comment (jchappelow) [inline node/consensus/engine.go:344]: What's the significance of this duration value?  Should this be the configured propose timeout, or is this something else?

File: node/services/jsonrpc/adminsvc/service.go (hunk 1 above)
Comment (charithabandi) [reply node/services/jsonrpc/adminsvc/service.go:598]: That makes sense

File: node/consensus/engine.go (hunk 2 above)
Comment (charithabandi) [reply node/consensus/engine.go]: blkPropTimer is what determines how quickly the node can catchup to the current consensus round (as this broadcasts the current block proposals), if this is pretty high, I observed that the block times are almost proportional to this.

Comment (charithabandi) [reply node/consensus/engine.go]: I will probably make this into a config

**Review by jchappelow — APPROVED** [review]
utACK.

**Review by jchappelow — APPROVED** [review]

**Pull request merged**